### ISSUE TITLE
About Training Accuracy Question 

### ISSUE THREAD

**Issue by jiandandian2** [mkocabas/VIBE#168]
1. operating system and the version: ubuntu18
2. python version: 3.6
3. pytorch version: 1.6

Did anyone else have the same problem? The accuracy is decreasing, and the error is increasing.

<img width="1920" alt="屏幕快照 2020-12-29 下午2 27 13" src="https://user-images.githubusercontent.com/33227739/103263156-fada1e80-49e1-11eb-9d50-61dc915afe3c.png">

Here is my train config:

GPU name -> GeForce RTX 2080 Ti
GPU feat -> _CudaDeviceProperties(name='GeForce RTX 2080 Ti', major=7, minor=5, total_memory=11019MB, multi_processor_count=68)
{'CUDNN': CfgNode({'BENCHMARK': True, 'DETERMINISTIC': False, 'ENABLED': True}),
 'DATASET': CfgNode({'SEQLEN': 16, 'OVERLAP': 0.5}),
 'DEBUG': False,
 'DEBUG_FREQ': 5,
 'DEVICE': 'cuda',
 'EXP_NAME': 'vibe',
 'LOGDIR': 'results/vibe_tests/02-12-2020_16-03-31_vibe',
 'LOSS': {'D_MOTION_LOSS_W': 0.5,
          'KP_2D_W': 300.0,
          'KP_3D_W': 300.0,
          'POSE_W': 60.0,
          'SHAPE_W': 0.06},
 'MODEL': {'TEMPORAL_TYPE': 'gru',
           'TGRU': {'ADD_LINEAR': True,
                    'BIDIRECTIONAL': False,
                    'HIDDEN_SIZE': 1024,
                    'NUM_LAYERS': 2,
                    'RESIDUAL': True}},
 'NUM_WORKERS': 8,
 'OUTPUT_DIR': 'results/vibe_tests',
 'SEED_VALUE': -1,
 'TRAIN': {'BATCH_SIZE': 32,
           'DATASETS_2D': ['Insta'],
           'DATASETS_3D': ['MPII3D'],
           'DATASET_EVAL': 'ThreeDPW',
           'DATA_2D_RATIO': 0.6,
           'END_EPOCH': 30,
           'GEN_LR': 5e-05,
           'GEN_MOMENTUM': 0.9,
           'GEN_OPTIM': 'Adam',
           'GEN_WD': 0.0,
           'LR_PATIENCE': 5,
           'MOT_DISCR': {'ATT': {'DROPOUT': 0.2,
                                 'LAYERS': 3,
                                 'SIZE': 1024},
                         'FEATURE_POOL': 'attention',
                         'HIDDEN_SIZE': 1024,
                         'LR': 0.0001,
                         'MOMENTUM': 0.9,
                         'NUM_LAYERS': 2,
                         'OPTIM': 'Adam',
                         'UPDATE_STEPS': 1,
                         'WD': 0.0001},
           'NUM_ITERS_PER_EPOCH': 500,
           'PRETRAINED': '',
           'PRETRAINED_REGRESSOR': 'data/vibe_data/spin_model_checkpoint.pth.tar',
           'RESUME': '',
           'START_EPOCH': 0}}
InstaVariety number of dataset objects 130431
MPII3D Dataset overlap ratio:  0
Loaded mpii3d dataset from data/vibe_db/mpii3d_train_db.pt
mpii3d - number of dataset objects 59934
AMASS dataset number of videos: 1012067
3DPW Dataset overlap ratio:  0.0
Loaded 3dpw dataset from data/vibe_db/3dpw_val_db.pt
3dpw - number of dataset objects 635
=> loaded pretrained model from 'data/vibe_data/spin_model_checkpoint.pth.tar'
 is not a pretrained model!!!!
=> no checkpoint found at ''

**Comment by jkamalu** [comment]
@jiandandian2 Did you ever end up resolving this? I see the exact same behavior.

**Comment by jiandandian2** [comment]
@jkamalu 
solved by adding the 3dpw training dataset
https://github.com/mkocabas/VIBE/issues/140

**Comment by yangshubin123** [comment]
How did you add the 3dpw? @jiandandian2

**Comment by sumingrui** [comment]
> @jkamalu
> solved by adding the 3dpw training dataset
> #140

I think the author dont want us use 3pdw.
[https://github.com/mkocabas/VIBE/issues/99#issuecomment-704368805](url)
